jobs/collectd/templates/collectd.d/write_riemann.conf.erb

```diff
@@ -0,0 +1,13 @@
+LoadPlugin "write_riemann"
```

Comment:
Maybe you should add if collectd.riemann_server empty so we don't load the plugin.